Procfile

```diff
@@ -1 +0,0 @@
-web: node keystone.js
```

Comment:
Removing this will kill our ability to deploy using heroku. Please serve the frontend if just static files via NGINX and proxy the backend requests to another dyno.

https://m.alphasights.com/using-nginx-on-heroku-to-serve-single-page-apps-and-avoid-cors-5d013b171a45